data/json/terrain.json

```diff
@@ -8453,5 +8453,225 @@
       "ter_set": "t_dirt",
       "items": [ { "item": "pipe", "count": [ 1, 2 ] }, { "item": "scrap", "count": [ 1, 40 ] } ]
     }
+  },
+  {
+    "type": "terrain",
+    "id": "t_ind_furnace",
+    "name": "industrial furnace",
+    "description": "Not the kind of furnace you'd heat your house with, this is a device for heating things to extreme temperatures as part of industrial crafting processes.",
+    "symbol": "0",
+    "color": "white_red",
+    "move_cost": 0,
+    "flags": [ "WALL", "BASHABLE", "PERMEABLE", "CONTAINER", "FIRE_CONTAINER", "SUPPRESS_SMOKE", "PLACE_ITEM" ],
+    "bash": {
+      "str_min": 30,
+      "str_max": 150,
+      "ter_set": "t_thconc_floor",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "cable", "charges": [ 0, 4 ] },
+        { "item": "scrap", "count": [ 8, 12 ] },
+        { "item": "steel_chunk", "count": [ 2, 4 ] },
+        { "item": "steel_plate", "count": [ 1, 2 ] }
+      ]
+    },
+    "deconstruct": {
+      "ter_set": "t_thconc_floor",
+      "items": [
+        { "item": "cable", "charges": [ 4, 8 ] },
+        { "item": "steel_chunk", "count": [ 4, 6 ] },
+        { "item": "steel_plate", "count": [ 2, 4 ] },
+        { "item": "sheet_metal", "count": [ 4, 6 ] },
+        { "item": "element", "count": [ 10, 25 ] },
+        { "item": "scrap", "count": [ 12, 16 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_ind_drill",
+    "name": "drill press",
+    "description": "A powerful drill mounted on a slide that lets it drop precisely down.",
+    "symbol": "7",
+    "color": "yellow_red",
+    "move_cost": 0,
+    "flags": [ "WALL", "PERMEABLE", "NOITEM", "BASHABLE", "DECONSTRUCT" ],
```

Comment:
`DECONSTRUCT` flag? AFAIK there is no such flag, I can't find any mention of it in source files, only in `JSON_FLAGS.md`. In `furniture.json` and `terrain.json` there are few occurrences of this flag. I suppose it is either outdated or non-existing.  Maybe you meant `EASY_DECONSTRUCT`?